Vostok.Hosting/Components/Log/LogLevelMetrics.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using Vostok.Commons.Time;
+using Vostok.Hosting.Abstractions;
+using Vostok.Logging.Abstractions;
+using Vostok.Metrics;
+using Vostok.Metrics.Models;
+using Vostok.Metrics.Primitives.Gauge;
+
+namespace Vostok.Hosting.Components.Log
+{
+    internal class LogLevelMetrics
+    {
+        private static readonly TimeSpan ScrapePeriod = 1.Minutes();
```

Comment:
Why should we hardcode scrape period instead of using whatever's set up in the provided `IMetricContext`?